tests/conftest.py

```diff
@@ -185,3 +188,21 @@ def message_status_post_body():
             }
         ]
     }
+
+@pytest.fixture
+def consumer():
+    with Session(database.engine()) as session:
```

Comment:
Not entirely sure whether this is right...I'm creating a "real" instance of a Consumer each time, as I couldn't find a way to stub or mock the session/db for unit tests. It also didn't seem like there was any automatic setup or teardown of a test db as I would expect, so the rows created persist over different test session, hence having to check if the Consumer exists already

Reply:
It's how we have written other database tests, we don't use test factories in this repo though we could and maybe should now we have a few different models. The test db gets truncated for each test and it's created/destroyed in CI so probably not an issue leaving like this for now. `factory-boy` is apparently pretty standard for Python and might help with associations etc.